tools/cli/internal/changelog/outputfilter/squash.go

```diff
@@ -150,6 +160,19 @@ func squashEntriesByValues(
 	return result, nil
 }
 
+func replaceOnlyFirstOccurrence(template, valuesToAddToTemplate string) string {
```

Comment:
[nit] can we add a comment somewhere on why we need to ensure replacing only the first occurrence?

Reply:
sure, I will address this comment in the next PR